.github/scripts/end2end/deploy-zenko.sh

```diff
@@ -142,4 +142,4 @@ for i in $(seq 1 120); do
     fi
     # Debug log to ease understanding of failures in the CI
     kubectl get pods -A
-done
+done
```

Comment:
nit: EOL should not be removed (and generally should not change files when not needed)